plansys2_planner/src/plansys2_planner/PlannerNode.cpp

```diff
@@ -90,6 +83,20 @@ PlannerNode::on_configure(const rclcpp_lifecycle::State & state)
       "POPF", "plansys2/POPFPlanSolver");
   }
 
+  get_plan_service_ = create_service<plansys2_msgs::srv::GetPlan>(
+    "planner/get_plan",
+    std::bind(
+      &PlannerNode::get_plan_service_callback,
+      this, std::placeholders::_1, std::placeholders::_2,
+      std::placeholders::_3));
+
+  validate_domain_service_ = create_service<plansys2_msgs::srv::ValidateDomain>(
+    "planner/validate_domain",
+    std::bind(
+      &PlannerNode::validate_domain_service_callback,
+      this, std::placeholders::_1, std::placeholders::_2,
+      std::placeholders::_3));
```

Comment:
Needed to move the service instantiations here since I ran into timing issues where another node could call these services before the planner was actually instantiated. I think this is good to fix regardless.